### PR TITLE
Use Builder

### PR DESCRIPTION
**Pull request by exogen**
Take the plunge. There are a few small code changes due to updated `eslint` rules.

/cc @ryan-roemer @boygirl

### CODE REVIEW

**Comment by ryan-roemer** [comment]
@exogen -- Lovely! LGTM.

**Comment by exogen** [comment]
On hold until we figure out FormidableLabs/builder#15 so I don't have to bump and release twice.

**Comment by exogen** [comment]
Updated to latest builder + archetype.

/cc @boygirl @ryan-roemer

File: package.json [hunk 1]
```diff
@@ -14,82 +14,26 @@
...
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
-    "babel-loader": "^5.3.2",
-    "d3": "^3.5.6",
-    "radium": "^0.14.1",
-    "rimraf": "^2.4.0",
-    "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "webpack": "^1.10.0"
+    "builder": "^2.1.0",
```
Comment (ryan-roemer) [inline package.json:23]: Open question -- we're not following semver for `v2`. Do we want to pin versions as `builder` evolves? Or just "go for it" and roll with the upgrade punches?

**Comment by ryan-roemer** [comment]
@exogen -- LGTM!

File: package.json (hunk 1 above)
Comment (exogen) [inline package.json:23]: I'm fine rolling with the punches myself. Breaking `builder` changes will most likely be discussed among the same people adopting it for now, anyway.

**Pull request merged**